docs/ReleaseNotes/ReleaseNotes.Atom.May.2019.md

```diff
@@ -0,0 +1,32 @@
+# Atom Release (May 2019)
+
+!!! warning
```

Comment:
Looks like the display of the text is not working well here. If you meant for dots in the start of each line, it is not displayed. Also, what is '!!!' ? I think it will be better to highlight it with ** warning **

Reply:
@MoriaYechiel that's fine; while the markdown preview in GitHub has some features, when we publish docs using mkDocs to github.io it turns out to look and feel just great.
_/* BTW: great minds think alike: I asked Eli if that `!!!` syntax is ok and he confirmed */_

Reply:
Cool! Didn't know